jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java

```diff
@@ -158,14 +158,14 @@ public InputStream getBlobInputStream(ResultSet result, String columnName)
         if (_dbName == null)
             throw new IllegalStateException("DbAdaptor missing metadata");
 
-        if (_dbName.startsWith("postgres"))
-        {
+        try {
+            Blob blob = result.getBlob(columnName);
+            return blob.getBinaryStream();
+        }
+        catch (SQLException ex) {
```

Comment:
I think this should catch "SQLFeatureNotSupportException", no?

Reply:
I was catching SQLException to also handle the 'postgres' case.
I changed it now to only catch  SQLFeatureNotSupportException and the 'postgres' case is now handled up-front as it was done before.